### ISSUE TITLE
Add Action to run tests on PR

### ISSUE THREAD

**Issue by C-Lodder** [whitew1994WW/film_night_discord_bot#18]
You've got an action that deploys to your AWS server when a PR is merged into `master`, but may be an idea to add a Github Action to run the tests in https://github.com/whitew1994WW/film_night_discord_bot/tree/master/tests when a PR is submitted/updated

**Comment by whitew1994WW** [comment]
Yes! If only the tests were up to date I could add this... cough... Stelios @kepler471
